test/testsuite/spectest.c

```diff
@@ -365,7 +365,9 @@ void usage(char *av0)
     fprintf(stdout, "\t-7\tAFP 3.4 version (default)\n");
     fprintf(stdout, "\t-v\tverbose\n");
     fprintf(stdout, "\t-V\tvery verbose\n");
-    fprintf(stdout, "\t-x\tdon't run tests that require special setup\n");
+#if 0
```

Comment:
Is there a reason you're just #if 0-ing this instead of just removing it completely, since you've unconditionally removed it from the usage string above?

Reply:
It's a minor thing, but it's meant for making it easy to bring the option back in the future while remembering to use the same letter code.

I can remove it for cleanliness sake!

Reply:
Removed. For future needs to skip tests, I say we introduce more fine-grained methods tied to certain capabilities.